examples/http_client.cr

```diff
@@ -0,0 +1,8 @@
+require "http"
```

Comment:
I think we can create specs to ensure the pool is generic.

Reply:
I had a feeling you might say that 🙂 I didn't know how to do it, though, so I figured I'd at least offer up what I had.

To clarify, I've always struggled with testing I/O. I don't know that we'd have to _actually_ test I/O here, but that hadn't crossed my mind when I was writing this.